src/catalog/backends/common/src/main/java/org/geoserver/cloud/event/remote/resourcepool/RemoteEventResourcePoolProcessor.java

```diff
@@ -72,10 +70,33 @@ public void onCatalogRemoteModifyEvent(CatalogInfoModified event) {
                         () -> log.trace("Ignoring event from self: {}", event));
     }
 
+    @EventListener(ResetEvent.class)
+    public void onReset(ResetEvent event) {
+        log.debug("Received a ResetEvent, triggering a GeoServer reset ({})", event);
```

Comment:
Looks like this log should be inside the `if` block?  otherwise it'll say `triggering a GeoServer reset event` when it won't